starter-kits/product/_overview_2.md

```diff
@@ -1,6 +1,6 @@
 % ## Cite this product
-
-% > Your citation
```

Comment:
This will still need the ">" blockquote symbol because this citation should be inside a blockquote; however, maybe we should word it something like this:
```
% Note: Must use a blockquote (">"), e.g.
% > Your citation
```

Reply:
Added back.